thunder/tests/framework.py

```diff
@@ -222,7 +238,7 @@ def version(self):
 
 def _all_test_executors():
     """Constructs a list of all Thunder executors to be used when generating tests."""
-    executors = [TorchExecutor]
+    executors = [TorchExecutor, TorchCompileCatExecutor, TorchCompileExecutor]
```

Comment:
Is this a bad idea? The CI has a lot of errors now. Is it worth working through each or should I instead propose this in a different PR?

Reply:
It's probably a bit much for the current PR